prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -209,10 +209,10 @@ ssize_t rxr_pkt_init_cts(struct rxr_ep *ep,
 		cts_hdr->flags |= RXR_CTS_READ_REQ;
 
 	/* CTS is sent by different communication protocols.
-	 * CTS is sent using tx_entry in the emulated longcts read 
-	 * protocol. The emulated longcts write and the longcts 
+	 * CTS is sent using tx_entry in the emulated longcts read
+	 * protocol. The emulated longcts write and the longcts
 	 * message protocols sends CTS using rx_entry.
-	 * This check ensures appropriate tx_id and rx_id are 
```

Comment:
Format change should be in a different commit

Reply:
done